### PR TITLE
Feature 502 create configurable bin dir

### PR DESCRIPTION
**Pull request by jprestop**
Created for Issue #502.  Testing using Travis CI build: https://www.travis-ci.org/github/NCAR/METplus/branches

### CODE REVIEW

**Review by georgemccabe — CHANGES_REQUESTED** [review]
I would recommend allowing MET_BIN_DIR to be unset or set to an empty string so that the MET tools can be found from the path (i.e. using modules to obtain MET). This would involve adding a default value in the calls to self.config.getdir, i.e.

`self.config.getdir('MET_BIN_DIR', '')`

I would also mention in the documentation that you can set MET_BIN_DIR to an empty string if the MET tools are found in the user's path, i.e. when using module load.

Also, it is a bit nit-picky, but I would suggest you line up the arguments to os.path.join. For example, in ascii2nc_wrapper.py line 33:

```
self.app_path = os.path.join(config.getdir('MET_BIN_DIR', ''),
                             self.app_name)
```

(remove space before self.app_name so it lines up with config)

**Comment by jprestop** [comment]
Thanks, George!  I made the requested changes. Please let me know if I need to do anything else.  I re-requested review.

File: docs/Contributors_Guide/create_wrapper.rst [hunk 1]
```diff
@@ -14,10 +14,10 @@ How to Create Your Own Wrapper
 
     def __init__(self, config, logger):
         super(NewToolWrapper, self).__init__(config, logger)
-        self.app_path = os.path.join(self.p.getdir('MET_INSTALL_DIR'), 'bin/new_tool')
+        self.app_path = os.path.join(self.p.getdir('MET_BIN_DIR'), 'new_tool')
```
Comment (georgemccabe) [inline docs/Contributors_Guide/create_wrapper.rst]: Add default value here.
```suggestion
        self.app_path = os.path.join(self.p.getdir('MET_BIN_DIR', ''),
                                     'new_tool')
```

File: parm/metplus_config/metplus_system.conf [hunk 2]
```diff
@@ -54,9 +54,12 @@ STAGING_DIR = {OUTPUT_BASE}/stage
 
 ##  Commonly used base MET variables
 
-## Met install location
+## MET install location
 MET_INSTALL_DIR = /path/to
 
+## MET binary directory
```
Comment (georgemccabe) [inline parm/metplus_config/metplus_system.conf:60]:
```suggestion
## MET binary directory
# Set this to an empty string to find the MET tools from the path
```

**Review by georgemccabe — CHANGES_REQUESTED** [review]
A couple minor changes, but otherwise I approve!

**Pull request merged**